src/Public/Responses.ps1

```diff
@@ -702,12 +697,13 @@ function Write-PodeJsonResponse {
         }
 
         'value' {
+            # Determine whether to use compression
+            $compress = -not $NoCompress
             if ($Value -isnot [string]) {
                 if ($Depth -le 0) {
-                    $Value = (ConvertTo-Json -InputObject $Value -Compress)
-                }
-                else {
-                    $Value = (ConvertTo-Json -InputObject $Value -Depth $Depth -Compress)
+                    $Value = (ConvertTo-Json -InputObject $Value -Compress:$compress)
+                } else {
+                    $Value = (ConvertTo-Json -InputObject $Value -Depth $Depth -Compress:$compress)
```

Comment:
You can just use `-Compress:(!$NoCompress)` to save the variable `$compress` assignment above 😄

Reply:
you are right let me change it